src/tests/gov/nasa/jpf/test/java/concurrent/ExchangerTest.java

```diff
@@ -63,7 +63,7 @@ public void testTimeoutExchange() {
       try {
         System.out.println("M now exchanging..");
 
-        String response = exchanger.exchange("there", 100, TimeUnit.MILLISECONDS);
```

Comment:
JPF uses virtual time, not real time, so changing the time setting does not affect the performance. Please revert this change (as we want to minimize changes to facilitate merging changes with various branches later).